communication-protocols/LoRaWAN/config.json

```diff
@@ -1,8 +1,8 @@
 {
     "fields": [
-        {"label":"DevEUI", "isRequired": true},
-        {"label":"AppEUI", "isRequired": true},
-        {"label":"AppKey", "isRequired": true}
+        {"label":"Dev EUI", "value":"devEui", "isRequired": true},

```

Comment:
In all our currently created devices, we used devE**UI** and appE**UI** (for some reason ...), better not change that